05-onchain.md

```diff
@@ -185,231 +161,192 @@ Similarly, when node A sees a *commitment transaction* from B:
+3. The recipient has committed to an outgoing HTLC for the offered one. In
+   this case the recipient has to use the preimage if they receive it from the
+   outgoing HTLC, otherwise they will lose funds by making an outgoing
    payment without redeeming the incoming one.
 
 
 ## Requirements
 
-
 If the node receives (or already knows) a payment preimage for an
 unresolved HTLC output it was offered for which it has committed to an
 outgoing HTLC, it MUST *resolve* the output by spending it.
 Otherwise, if the other node is not irrevocably committed to the HTLC,
 it MUST NOT *resolve* the output by spending it.
 
-
 To spend an offered HTLC output: if the transaction is the node's own commitment transaction, then it MUST use the HTLC-success transaction, and the
 HTLC-success transaction output MUST be *resolved* as described in
 "On-chain HTLC Transaction Handling", otherwise, it MUST spend the output to a convenient address.
 
-
 If not otherwise resolved, once the HTLC output has expired, it is considered
 *irrevocably resolved*.
 
-
 ## Rationale
 
-
-If this is our commitment transaction, we can only use a payment
+If this is the recipient's commitment transaction, they can only use a payment
 preimage with the HTLC-success transaction (which preserves the
-`to_self_delay` requirement).  Otherwise we can create any transaction we want to
+`to_self_delay` requirement). Otherwise the recipient can create any transaction they want to
 resolve it.
 
-
-We don't care about expired offers: we should have ensured that the
+The recipient doesn't care about expired offers: they should have ensured that the
 HTLC can only expire long it is needed.
 
-
 # On-chain HTLC Transaction Handling
 
-
 Because to-self payments have to be delayed (to allow time for a
-penalty transaction), HTLC outputs can only be spent by the node which
+penalty transaction), HTLC outputs can only be spent by the node that
 broadcast the *commitment transaction* using the HTLC-timeout or the
 HTLC-success transaction, which include that delay.
 
-
 ## Requirements
 
-
 A node SHOULD resolve its own HTLC transaction output by spending it
-to a convenient address.  A node MUST wait until the
+to a convenient address. A node MUST wait until the
 `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified by the other
 node's `open_channel` `to_self_delay` field) before spending the
 output.
 
-
 If the output is spent (as recommended), the output is *resolved* by
 the spending transaction, otherwise it is considered *resolved* by the
 *commitment transaction* itself.
 
-
 ## Rationale
 
-
 Spending the `to_local` output avoids having to remember the complicated
 witness script associated with that particular channel for later
 spending.
 
-
 # Revoked Transaction Close Handling
 
-
-If a node tries to broadcast old state, we can use the revocation key
+If a node tries to broadcast old state, its peer can use the revocation key
```

Comment:
...state, the other node can...

'peer' refers specifically to neighboring nodes, as defined in BOLT 4

Reply:
Changed. @toadlyBroodle, you should double-check that the definitions for peer and node in BOLT-4 match with what I wrote for BOLT-0. I think it's in the master branch now.

Reply:
Ok thanks, I will have a look at BOLT 0 definition.